RadarSDK/RadarUtils.m

```diff
@@ -69,6 +69,37 @@ + (BOOL)locationBackgroundMode {
     return backgroundModes && [backgroundModes containsObject:@"location"];
 }
 
++ (NSString *)locationAuthorization {
+    CLAuthorizationStatus authorizationStatus = CLLocationManager.authorizationStatus;
+    switch (authorizationStatus) {
+    case kCLAuthorizationStatusAuthorizedWhenInUse:
+        return @"GRANTED_FOREGROUND";
+    case kCLAuthorizationStatusAuthorizedAlways:
+        return @"GRANTED_BACKGROUND";
+    case kCLAuthorizationStatusDenied:
+        return @"DENIED";
+    case kCLAuthorizationStatusRestricted:
+        return @"DENIED";
```

Comment:
Should this be `@"RESTRICTED"`?

Reply:
We'll just treat restricted as `DENIED` so we don't have to support a separate enum val on the server. It's sort of a rare state that means "denied with parental controls turned on" https://developer.apple.com/documentation/corelocation/clauthorizationstatus/restricted